src/main/java/com/vmware/vipclient/i18n/VIPCfg.java

```diff
@@ -192,10 +193,8 @@ public void initialize(String cfg) throws VIPClientInitException {
      * initialize VIPService instances to provide HTTP requester
      */
     public void initializeVIPService() {
```

Comment:
Add @deprecated onto this method?

Reply:
I have also removed all calls to this method from tests.